mistletoe/block_token.py

```diff
@@ -19,7 +19,8 @@
 Tokens to be included in the parsing process, in the order specified.
 """
 __all__ = ['BlockCode', 'Heading', 'Quote', 'CodeFence', 'ThematicBreak',
-           'List', 'Table', 'Footnote', 'Paragraph']
+           'List', 'Table', 'Footnote', 'DefinitionList',
+           'Paragraph', 'DefinitionTerm', 'DefinitionDesc']
```

Comment:
Actually, we don't need / shouldn't put `DefinitionTerm` and `DefinitionDesc` into the parsing process, as they are to be instantiated only from the `DefinitionList.read()` method, right?

Reply:
You are right, I removed them.